### ISSUE TITLE
Is this project Dead ?

### ISSUE THREAD

**Issue by royalterra** [christian-putzke/Roundcube-CardDAV#98]
Here is many pull request and not one reply to issues is the project dead

**Comment by christian-putzke** [comment]
Yep it is dead. I haven't been working on the plugin since years and it's totally outdated. Maybe I'll just make a last commit which addresses this fact and has some links to alternatives ;)

**Comment by IzzySoft** [comment]
@christian-putzke would you still do that last commit? Would be good to know

* what is the last version of Roundcube this works with
* what alternatives could one switch to ([carddavsso](https://github.com/Windkracht8/carddavsso) seems to be closest; [roundcube-select_for_sync](https://github.com/syncgw/roundcube-select_for_sync) has quite a few additional dependencies)

Thanks!
